custom-dockerfiles/neutron-server/Dockerfile

```diff
@@ -5,26 +5,18 @@ WORKDIR /tmp
 
 RUN sed -i 's/$stream/8-stream/g' /etc/yum.repos.d/CentOS-* && \
         yum -y install git && \
-	git clone https://github.com/ansible-network/network-runner && \
+	git clone https://github.com/CCI-MOC/network-runner && \
 	cd network-runner && \
-        git checkout 0.3.6 && \
+        git checkout esi_network_runner && \
 	python3 setup.py install && \
 	yum clean all
 
-RUN cd /tmp && \
-        mkdir naved && \
-        cd naved && \
-        git clone https://github.com/naved001/network-runner && \
-        cd network-runner && \
-        git checkout vdx-nos/improve-performance && \
-        \cp -r etc/ansible/roles/network-runner/providers/nos /etc/ansible/roles/network-runner/providers/.
-
 RUN cd /tmp && \
         mkdir tzumainn && \
         cd tzumainn && \
```

Comment:
Ah, we can probably use a more neutral directory name now - something like `esi` is probably good enough!

Reply:
Done!